adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java

```diff
@@ -141,8 +141,12 @@ public void login(PrintStream printer, Reader reader) throws IOException, Server
     }
 
     public void logout() throws IOException, InterruptedException, URISyntaxException {
+        logout(logoutUrl -> Desktop.getDesktop().browse(new URI(logoutUrl)));
+    }
+
+    public void logout(BrowseCallback browseCallback) throws IOException, InterruptedException, URISyntaxException {
```

Comment:
It would be better to have a method to setBrowseCallback on KeycloakInstalled instead of overloading method

Reply:
Fixed in https://github.com/keycloak/keycloak/pull/7747